src/instructlab/sdg/llmblock.py

```diff
@@ -189,7 +189,7 @@ def generate(self, samples: Dataset) -> Dataset:
         valid_samples = []
 
         for sample in samples:
-            if self._validate(self.prompt_template, sample):
+            if self.validate(self.prompt_template, sample):
```

Comment:
This looks like a bugfix for ConditionalLLMBlock, orthogonal to these new blocks

I've filed #186 - let's remove it from this PR